test/integration/services/lnd/utils.spec.ts

```diff
@@ -120,4 +124,9 @@ describe("lndUtils", () => {
 
     expect((endBalance - initBalance) * 1000).toBeCloseTo(totalFees, 0)
   })
+
+  it("runs deleteExpiredInvoiceUser without throw an exception", async () => {
+    const result = await deleteExpiredInvoiceUser()
+    expect(result.deletedCount).toBe(0)
```

Comment:
Maybe we can mock the date and test whether it actually deletes the invoices?

Reply:
Fixed